database/migrations/2022_03_02_145348_create_track_routes_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateTrackRoutesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('track_routes', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('user_id')->nullable();
```

Comment:
Why do not maked a foreign key?

Reply:
```suggestion
                $table->foreign('user_id')->references('id')->on('users');
```

seems thgat casade delete is missing https://laravel.com/docs/9.x/migrations#foreign-key-constraints 

```php
$table->foreignId('user_id')
      ->constrained()
      ->onUpdate('cascade')
      ->onDelete('cascade');
```